Cargo.lock

```diff
@@ -77,9 +77,9 @@ dependencies = [
 
 [[package]]
 name = "bumpalo"
-version = "3.11.1"
+version = "3.12.0"
```

Comment:
Removed Cargo.lock as it is not recommended for a library: see answer of `alexcrichton` https://github.com/rust-lang/cargo/issues/315#issuecomment-51075730

This way, dependencies are not locked and are not scanned/reported by Github Security Alert